lib/cocoapods/validator.rb

```diff
@@ -519,8 +519,11 @@ def parse_xcodebuild_output(output)
     #         returns its output (both STDOUT and STDERR).
     #
     def xcodebuild
-      UI.puts 'xcodebuild clean build -target Pods' if config.verbose?
-      output = `xcodebuild clean build -target Pods 2>&1`
+      command = 'xcodebuild clean build -target Pods CODE_SIGN_IDENTITY=-'
```

Comment:
`CODE_SIGN_IDENTITY` set to `-` so that device builds will fail even if one has it all set up locally.